client/app/account/user/advanced/user-advanced.html

```diff
@@ -1,85 +1,75 @@
-                                  aria-hidden="true">
-                            </span>
-                            <div data-ng-bind-html="'user_account_advanced_section_developer_api' | translate "></div>
-                        </div>
-                    </div>
+        <div data-ovh-alert="useraccount.alerts.dashboardAdvanced"></div>
 
-                    <div class="oui-box oui-box_light">
-                        <div class="oui-switch oui-switch_inline">
-                            <label class="oui-switch__label-container">
-                                <input type="checkbox"
-                                       class="oui-switch__checkbox"
-                                       data-ng-model="advancedCtrl.developmentMode.enabled"
-                                       data-ng-change="advancedCtrl.updateDevelopmentMode()"
-                                       data-ng-disabled="advancedCtrl.isLoadingDeveloperMode"
-                                       data-track-on="click"
-                                       data-track-type="action"
-                                       data-track-name="account::advanced_settings::edit::developer_mode">
-                                <span class="oui-switch__slider" aria-hidden="true"></span>
-                            </label>
-                        </div>
-                        <span data-translate="user_account_advanced_section_developer_label"></span>
-                    </div>
+        <h2 data-translate="user_account_advanced_section_developer_title"></h2>
 
-                    <h3 class="mt-5"
-                        data-translate="user_account_advanced_section_developer_join_us">
-                    </h3>
-
-                    <ul class="follow-link-list list-unstyled my-5">
-                        <li class="mb-2">
-                            <span class="fa fa-2x fa-github text-center mr-2"
-                                  aria-hidden="true">
-                            </span>
-                            <a target="_blank"
-                               href="https://github.com/ovh-ux"
-                               class="align-bottom">
-                                https://github.com/ovh-ux
-                            </a>
-                        </li>
-                        <li class="mb-2">
-                            <span class="fa fa-2x fa-comments text-center mr-2"
-                                  aria-hidden="true">
-                            </span>
-                            <a target="_blank"
-                               href="https://gitter.im/ovh/ux"
-                               class="align-bottom">
-                                https://gitter.im/ovh/ux
-                            </a>
-                        </li>
-                    </ul>
-                </form>
+        <form>
+            <div class="row my-5">
+                <div class="col-xs-12 col-md-4 text-center">
```

Comment:
`col-xs-12` aren't needed.

```suggestion
                <div class="col-md-4 text-center">
```